mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -20,6 +20,7 @@ CLASS sg net/minecraft/entity/Entity
 	FIELD N prevRenderY D
 	FIELD O prevRenderZ D
 	FIELD P stepHeight F
+	FIELD Q noClip Z
```

Comment:
Does MC's no clipping affect entity collision as well? If so, I think `ignoreBlockCollision` may be a better name.